Validation/verify_demo.ipynb

```diff
@@ -0,0 +1,401 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "toc-hr-collapsed": false
+   },
+   "source": [
+    "# Quickstart LSST Verify Demo\n",
+    "\n",
+    "<br>Owner: **Keith Bechtol** ([@bechtol](https://github.com/LSSTScienceCollaborations/StackClub/issues/new?body=@bechtol))\n",
+    "<br>Last Verified to Run: **v17.0**\n",
+    "<br>Verified Stack Release: **v17.0**, **d_2019_02_04**, **d_2019_01_15**\n",
+    "\n",
+    "This notebook demonstrates basic functionality of the LSST Verify python pakcage: https://github.com/lsst/verify . The notebook is based on the documentation at https://sqr-019.lsst.io/ . \n",
+    "\n",
+    "Another example from the LSST Systems Engineering team can be found [here](https://github.com/mareuter/notebooks/blob/master/LSST/Systems_Engineering/System_Verification_SQuaSH/System_Verification_Demo.ipynb), for which the metrics are defined [here](https://github.com/mareuter/notebooks/tree/master/LSST/Systems_Engineering/System_Verification_SQuaSH).\n",
+    "\n",
+    "### Learning Objectives\n",
+    "\n",
+    "After working through and studying this notebook you should be able to\n",
+    "   1. Create custom metrics for your science cases and associate multiple specifications with those metrics\n",
+    "   2. Evaluate metrics and store the output for subsequent analysis\n",
+    "   3. Create customized summary displays for the performance on those metrics relative to your specifications\n",
+    "\n",
+    "### Logistics\n",
+    "\n",
+    "This notebook is intended to be runnable on `lsst-lsp-stable.ncsa.illinois.edu` from a local git clone of https://github.com/LSSTScienceCollaborations/StackClub.\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# What version of the Stack am I using?\n",
+    "! echo $HOSTNAME\n",
+    "! eups list -s | grep lsst_distrib"
```

Comment:
You can just say `eups list -s lsst_distrib`.  You need not grep for the package.

Reply:
Thanks. Done.